sites/gent/ch_hpc_policies.tex

```diff
@@ -1,4 +1,85 @@
 \chapter{HPC Policies}
 \label{ch:hpc-policies}
 
-Stub chapter
+Everyone can get access and use the HPC-UGent supercomputing infrastructure and services.
+The conditions that apply depend on your affiliation.
+
+\section{Access for staff & academics}
+
+\subsection{Researchers and staff affiliated with Flemish university associations}
+
+\begin{itemize}
+  \item Includes externally funded researchers registered in the personnel database (FWO, SBO, VIB, IMEC, etc.)
+  \item Includes researchers from all VSC partners
+  \item Usage is free of charge
+  \item Use your account credentials at your affiliated university to request a VSC-id and connect
+\end{itemize}
+
+
+\subsection{Researchers and staff affiliated with other Flemish or federal research institutes}
+
+\begin{itemize}
+  \item Includes researchers from e.g. INBO, ILVO, RBINS, etc.
+  \item Usage subject to an agreement with Ghent University
+  \item Every institute gets 1500 euro worth of compute time free of charge per year (corresponds to e.g. 90.000 core hours on cluster golett)
+  \item Use your 'welkom' account credentials to request a VSC-id and connect
+\end{itemize}
+
+
+\subsection{Students}
+
+\begin{itemize}
+  \item Students can also use HPC-UGent (Bachelor or Master, enrolled in institute mentioned above)
+  \item Same conditions apply, free of charge for all Flemish university associations
+  \item Use your university account credentials to request a VSC-id and connect
+\end{itemize}
+
+
+
+\section{Access for industry}
+
+Researchers and developers from industry are very welcome to use HPC-UGent services and infrastructure.
+
+
+\subsection{Our offer}
+
+\begin{itemize}
+  \item Access to state-of-the-art supercomputing infrastructure
+  \item Dedicated helpdesk and user support
+  \item On-demand software installation
+  \item Regular training sessions
+  \item Buy compute time in pay-what-you-use model
+  \item Advice on using other compute clusters in the VSC network
+\end{itemize}
+
+
+\subsection{Access models}
+
+\begin{itemize}
+  \item Exploratory access
+  \begin{itemize}
+    \item Explore our infrastructure and services for a 1-month period
+    \item No contract needed
+    \item Free of charge
+    \item Request your free trial by mail to hpc@ugent.be
+  \end{itemize}
+  \item Access via agreement
+  \begin{itemize}
+    \item Subject to an agreement between your company and Ghent University
+    \item Check out a draft contract at https://www.ugent.be/hpc/en/industry/industry
+    \item Pay-what-you-use system: you only pay for the compute time that you actually use
+    \item Pricing starts at 0.015 euro / core hour (i.e. for using 1 core for 1 hour)
```

Comment:
`using 1 processor core`

Reply:
I removed (outdated) pricing here altogether.
The pricing is mentioned in the draft contract
(and they can always get a quote by mail)